core/models.py

```diff
@@ -95,8 +97,8 @@ class Operation(BaseEntity):
     sample_output_path = models.TextField(blank=True, null=True)
     tags = models.ManyToManyField(Tag)
     is_draft = models.BooleanField(default=True)
-    row_count = models.IntegerField(blank=True, null=True)
-    count_rows = models.BooleanField(default=False) # controls whether to count rows in the post_save signal
```

Comment:
The reason these were removed? This is going to break more than a few things

Reply:
Removed